bin/dbt-build.py

```diff
@@ -0,0 +1,355 @@
+#!/usr/bin/env python3
+
+import os
+DBT_ROOT=os.environ["DBT_ROOT"]
+exec(open(f'{DBT_ROOT}/scripts/dbt_setup_constants.py').read())
+
+import sys
+sys.path.append(f'{PROD_BASEPATH}/dunedaq-v2.8.2/dbt-pyvenv/lib/python3.8/site-packages')
```

Comment:
This script is supposed to be called within the build environment. So `dbt-workarea-env` must have been run already, and `DBT_AREA_ROOT` is available together with the `dbt-pyvenv` subdirectory. 

I suggest to changing it to `sys.path.append(f'{DBT_AREA_ROOT}/dbt-pyvenv/lib/python3.8/site-packages')`, and moving the `find_work_area()` call and check to be prior to this.